elasticdl/python/master/k8s_instance_manager.py

```diff
@@ -71,10 +74,14 @@ def __init__(
         self._relaunch_deleted_live_ps = True
 
         self._k8s_client = k8s.Client(event_callback=self._event_cb, **kwargs)
-        self._ps_addrs = self._get_ps_addrs()
+        self._ps_addrs = self._get_addrs(
```

Comment:
Why not store _ps_addrs as a string array instead of a string concatenated with separator?

Reply:
Because we need to pass this as a string to `worker.main`.